pytket/extensions/aqt/multi_zone_architecture/circuit/multizone_circuit.py

```diff
@@ -366,6 +380,26 @@ def copy(self) -> "MultiZoneCircuit":
         new_circuit.multi_zone_operations = deepcopy(self.multi_zone_operations)
         return new_circuit
 
+    def get_n_shuttles(self) -> int:
```

Comment:
Were are this two functions used?

Reply:
Just there for characterization of circuit after compilation. I added some test code for it.